src/main/resources/interface-specification.yml

```diff
@@ -451,6 +451,114 @@ paths:
           schema:
             type: boolean
             default: false
+  '/agreements/filter/producers':
```

Comment:
the path `/agreements/producers` cannot be used because it clashes with `/agreements/{agreementId}`.
Do you have any suggestion to improve this name? @beetlecrunch @TonioGela

Reply:
`/agreements/{agreementId}/producers` is a no-go since we need to get the producers for a bunch of agreements together right? Another option, since we're getting producers and not agreements is to invert the tokens in the path, and ask for `/producers?agreements=yada,yada`. How about that?

Reply:
correct, the first suffers from that limitation.
But I actually like your second solution, it doesn't have the `agreements` prefix but we are already in the agreement service, there's no need for more context 🚀

Reply:
I like the second solution too.
I was going to suggest `/filters/agreements/producers`, but @TonioGela  is more elegant and clean.